evalml/pipelines/components/estimators/classifiers/kneighbors_classifier.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from sklearn.neighbors import KNeighborsClassifier as SKKNeighborsClassifier
+from skopt.space import Integer
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+
+
+class KNeighborsClassifier(Estimator):
+    """
+    K-Nearest Neighbors Classifier.
+    """
+    name = "K-Nearest Neighbors Classifier"
+    hyperparameter_ranges = {
+        "n_neighbors": Integer(2, 12),
+        "weights": ["uniform", "distance"],
+        "algorithm": ["auto", "ball_tree", "kd_tree", "brute"],
+        "leaf_size": Integer(10, 30),
+        "p": Integer(1, 5)
+    }
+    model_family = ModelFamily.K_NEIGHBORS
+    supported_problem_types = [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]
```

Comment:
I think this estimator would work for time series problems?

Reply:
Got it, added.